backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -817,6 +817,8 @@ public QueueArguments.Builder workQueueArgumentsBuilder() {
         if (useQuorumQueues) {
             builder.quorumQueue().replicationFactor(quorumQueueReplicationFactor);
             quorumQueueDeliveryLimit.ifPresent(builder::deliveryLimit);
+        } else {
+            builder.put("x-queue-version", 2);
```

Comment:
Its inter-compatibility with the older James versions is good.
This works (be skipped) with old RabbitMQ version 3.8.x and 3.9.x for example.